src/commands/init.js

```diff
@@ -140,6 +142,122 @@ async function init() {
   progress.succeed(messages.generatedConfig());
 }
 
+const sleep = (time: number = 1000) =>
+  new Promise(resolve => setTimeout(resolve, time));
+
+/**
+ * Adds Haul to native iOS build pipeline
+ */
+const addToXcodeBuild = async (cwd: string) => {
+  let entry;
+
+  // Does `ios/*.xcodeproj` exist?
+  const iosPath = path.join(cwd, 'ios');
+  if (fs.existsSync(iosPath)) {
+    const list = fs
+      .readdirSync(path.join(cwd, 'ios'))
+      .filter(file => file.includes('.xcodeproj'));
+
+    // Do nothing if multiple projects were found
+    if (list.length === 1) {
+      entry = path.join(iosPath, list[0]);
+    }
+  }
+
+  // Otherwise, ask for path to a file
+  if (!entry) {
+    const result = await inquirer.prompt([
+      {
+        type: 'input',
+        name: 'entry',
+        message: 'Enter path to the .xcodeproj file',
+        validate: pathToFile =>
+          fs.existsSync(pathToFile) && pathToFile.includes('.xcodeproj')
+            ? true
+            : `${pathToFile} is not a valid .xcodeproj`,
+      },
+    ]);
+
+    entry = path.resolve(cwd, result.entry);
+  }
+
+  const progress = ora('Adding haul to your native build pipeline');
```

Comment:
`Adding haul to your XCode build scripts`

Though I feel like we should ask before this change. Looks rather invasive and I might just want to try before integrating.